common/app/routes/challenges/components/Classic-Modal.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { PropTypes } from 'react';
+import { Button, Modal } from 'react-bootstrap';
+import PureComponent from 'react-pure-render/component';
+import FontAwesome from 'react-fontawesome';
+
+import { randomCompliment } from '../../../utils/get-words';
+
+export default class ClassicModal extends PureComponent {
+
+  renderIcon() {
+    return (
```

Comment:
A single element render does not need to be in it's own render method.

We usually want to move element into their own render methods when there is logic involved (condition rendering or mapping over an array) or when it is clear that the element is almost it's own component.